contracts/Bank.sol

```diff
@@ -325,4 +337,38 @@ contract Bank is BankStorage, Ownable, Initializable {
         if (_value > 0) return (true, _value, _time);
         return (false, 0, _time);
     }
+
+    /**
+     * @dev Allows admin to add address to keeper role
+     * @param keeper address of new keeper
+     */
+    function addKeeper(address keeper) external {
+        require(keeper != address(0), "operation not allowed");
+        grantRole(KEEPER_ROLE, keeper);
+    }
+
+    /**
+     * @dev Allows admin to remove address from keeper role
+     * @param oldKeeper address of old keeper
+     */
+    function revokeKeeper(address oldKeeper) external {
+        revokeRole(KEEPER_ROLE, oldKeeper);
+    }
+
+    /**
+     * @dev Allows admin to add address to price updater role
+     * @param updater address of new price updater
+     */
+    function addPriceUpdater(address updater) external {
+        require(updater != address(0), "operation not allowed");
+        grantRole(PRICE_UPDATER_ROLE, updater);
+    }
+
+    /**
+     * @dev Allows admin to remove address from price updater role
+     * @param oldUpdater address of old price updater
+     */
+    function revokePriceUpdater(address oldUpdater) external {
+        revokeRole(PRICE_UPDATER_ROLE, oldUpdater);
```

Comment:
Can you change `PRICE_UPDATER` to `REPORTER`, sticking with Tellor terminology for that role